products/bigquery/inspec.yaml

```diff
@@ -29,12 +29,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         exclude_plural: true
       name: !ruby/object:Overrides::Inspec::PropertyOverride
         exclude_plural: true
+        exclude_reader: true
       lastModifiedTime: !ruby/object:Overrides::Inspec::PropertyOverride
         exclude_plural: true
+    additional_functions: |
```

Comment:
Might be overkill, but what do you think about either:

1) Having additional_functions take in a list of Ruby files
2) Using the compile() function to add a Ruby file.

That way, you can put this Ruby code in a Ruby file that IDEs, Rubocop + the like can test?

Reply:
Moved it to using compile